treeherder/client/thclient/client.py

```diff
@@ -701,6 +711,16 @@ def _post_json(self, project, endpoint, oauth_key, oauth_secret, jsondata,
                              timeout=timeout)
         resp.raise_for_status()
 
+    def get_revision_jobs(self, repo_name, revision):
+        """
+        Return all the jobs for a given revision from treeherder
+        """
+        revision_id = self._get_revision_id(repo_name, revision)
+        url = "%s/%s/jobs/?count=2000&result_set_id=%s" \
```

Comment:
Also, all the same comments on get_revision_id (repo_name vs. project, _get_uri, endpoint defining, json parsing) apply here.